src/Map/src/Renderer/AbstractRenderer.php

```diff
@@ -108,6 +108,14 @@ private function getMapAttributes(Map $map): array
             $attrs['polylines'][$key]['@id'] = $computeId($polyline);
         }
 
+        foreach ($attrs['circles'] as $key => $circle) {
+            $attrs['circles'][$key]['@id'] = $computeId($circle);
+        }
+
+        foreach ($attrs['rectangles'] as $key => $rectangle) {
+            $attrs['rectangles'][$key]['@id'] = $computeId($rectangle);
+        }
+
```

Comment:
The `@id` computation was missing from last PR and this one, it was not possible to use them correctly when using the Map with a LiveComponent. 

I've improved our tests to assert that `@id` are present and nicely computed, for the next `Elements` instances properties.